src/components/tutorsCard.js

```diff
@@ -2,25 +2,25 @@ import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import { faStar } from '@fortawesome/free-solid-svg-icons';
 import '../assets/styles/components/tutorCards.scss';
 
-function TutorsCard(props) {
-  const tutors = props;
```

Comment:
You can destructuring tutor.props and use only rating instead of tutor.props.rating. Same with others elements (name, profession, …)

Reply:
You're right. It's done.